tests/console/consoletest_setup.pm

```diff
@@ -39,31 +39,22 @@ sub run {
     # Stop packagekit
     systemctl 'mask packagekit.service';
     systemctl 'stop packagekit.service';
-    # Installing a minimal system gives a pattern conflicting with anything not minimal
-    # Let's uninstall 'the pattern' (no packages affected) in order to be able to install stuff
-    script_run 'rpm -qi patterns-openSUSE-minimal_base-conflicts && zypper -n rm patterns-openSUSE-minimal_base-conflicts';
-    # Install curl and tar in order to get the test data
-    zypper_call 'install curl tar';
-
     # upload_logs requires curl, but we wanted the initial state of the system
     upload_logs "/tmp/psaxf.log";
     upload_logs "/tmp/loadavg_consoletest_setup.txt";
 
-    # BSC#997263 - VMware screen resolution defaults to 800x600
-    if (check_var('VIRSH_VMM_FAMILY', 'vmware')) {
-        assert_script_run("sed -ie '/GFXMODE=/s/=.*/=1024x768x32/' /etc/default/grub");
-        assert_script_run("sed -ie '/GFXPAYLOAD_LINUX=/s/=.*/=1024x768x32/' /etc/default/grub");
-        assert_script_run("grub2-mkconfig -o /boot/grub2/grub.cfg");
-    }
-
```

Comment:
I do not know for sure why you move this part into system_prepare but not the next step but I am sure you have your reasons :)

Reply:
My assumption is that this step is required to be done once, the next step makes sense every time we have logs. I do know that we don't have such scenario where it would be called not after installation only, but this part we may change, also by scheduling module multiple times, e.g. after reboot.